java-backend/src/main/java/org/kframework/backend/java/symbolic/JavaBackend.java

```diff
@@ -118,7 +118,8 @@ public JavaBackend(KExceptionManager kem, FileUtil files, GlobalOptions globalOp
         ModuleTransformer convertDataStructureToLookup = ModuleTransformer.fromSentenceTransformer((m, s) -> new ConvertDataStructureToLookup(m, false).convert(s), "convert data structures to lookups");
 
         //Commented steps are (possibly) required for spec automaton generation.
-        return m -> ModuleTransformer.fromSentenceTransformer(new ResolveAnonVar()::resolve, "resolve anonymous varaibles")
+        return m -> ModuleTransformer.fromSentenceTransformer(Kompile::removePolyKLabels, "remove poly klabels")
```

Comment:
I guess the plan is to just not support definitions with parametric productions on the Java backend?

Reply:
Well, they're supported inasmuch as they were before, we're simply not actually inserting parameters into the klabels that the backend sees.